docs/admin/federation/index.md

```diff
@@ -137,11 +137,11 @@ cluster.
 Assuming our YAML file is located at `/cluster1/cluster.yaml`, we can run the following command to register this cluster:
 
 ```shell
-$ kubectl create -f /cluster1/cluster.yaml --cluster=federation-cluster
+$ kubectl create -f /cluster1/cluster.yaml --context=federation-cluster
```

Comment:
Added a TODO.

Filed an issue here - https://github.com/kubernetes/kubernetes/issues/30394

Btw, not sure what's the confusion in Kelsey's tutorial. It correctly uses --`context=federation-cluster` in the commands - https://github.com/kelseyhightower/kubernetes-cluster-federation/tree/4c581e4eb6693186ddd19cd523e64cc7f9a9f35d